srvmon

```diff
@@ -220,6 +223,9 @@ cleanupOKCheckFiles() {
 
 
 writeHeader() {
+  if [[ "$cgi" = true ]]; then
```

Comment:
Please use variable expansion with curly brackets to remain the same style as the rest of the script ( ${cgi} ). Also, does the single = work? (That would be new to me, I'd expect it to be $cgi == "true", but hey you keep learning every day)

Reply:
Done.

Like, all things, bash is weird. `[[` is a upgrade for `[`, which is almost an alias for `test`, which basically interprets the arguments on it's own (single `=` included). 

`==` is actually only supported on `[[`. It fails on both `[` and `test`, which simply use `=` for equality. This is also supported by `[[`.

Reply:
You can do these and variations on the command line to get a feeling

`test 1 = 2`
`[ 1 = 2 ]`
`[[ 1 = 2 ]]`

Reply:
Thanks you for that explanation. I was unaware of that. Almost all of my scripts target recent versions of bash, not sh or other Unixes.